eth-rpc-adapter/src/eip1193-bridge.ts

```diff
@@ -256,6 +256,17 @@ class Eip1193BridgeImpl {
     return hexValue(val);
   }
 
+  async eth_getEthGas(params: any[]): Promise<{
+    gasPrice: string;
+    gasLimit: string;
+  }> {
+    const res = await this.#provider._getEthGas(params[0], params[1], params[2]);
+    return {
+      gasPrice: res.gasPrice.toString(),
+      gasLimit: res.gasLimit.toString(),
```

Comment:
```suggestion
      gasPrice: hexValue(res.gasPrice),
      gasLimit: hexValue(res.gasLimit),
```

Keep the same as `eth_estimateGas`